src/puppetlabs/puppetdb/middleware.clj

```diff
@@ -41,25 +41,25 @@
     (log/debug (trs "Processing HTTP request to URI: ''{0}''" (:uri req)))
     (app req)))
 
-(defn build-whitelist-authorizer
+(defn build-allowlist-authorizer
   "Build a function that will authorize requests based on the supplied
-  certificate whitelist (see `cn-whitelist->authorizer` for more
+  certificate allowlist (see `cn-whitelist->authorizer` for more
   details). Returns :authorized if the request is allowed, otherwise a
   string describing the reason not."
-  [whitelist]
-  {:pre  [(string? whitelist)]
+  [allowlist]
+  {:pre  [(string? allowlist)]
    :post [(fn? %)]}
-  (let [allowed? (kitchensink/cn-whitelist->authorizer whitelist)]
+  (let [allowed? (kitchensink/cn-whitelist->authorizer allowlist)]
```

Comment:
Do we need to do an X release of kitchensink?

Reply:
I think we will need a release of kitchensink to get rid of this func name.